common/lib/common-utils/src/logger.ts

```diff
@@ -10,7 +10,7 @@ import {
 	ITelemetryGenericEvent,
 	ITelemetryLogger,
 	ITelemetryPerformanceEvent,
-} from "@fluidframework/common-definitions";
+} from "@fluidframework/core-interfaces";
```

Comment:
Tying to Tony's comment, this import needs to keep using common-definitions for now, until/if these implementations move somewhere else.